examples/medplum-patient-intake-demo/src/components/PatientDetails.tsx

```diff
@@ -20,6 +22,8 @@ export function PatientDetails(props: PatientDetailsProps): JSX.Element {
     ['details', 'Details'],
     ['edit', 'Edit'],
     ['history', 'History'],
+    ['observations', 'SDOH Observations'],
+    ['consents', 'Consents'],
```

Comment:
Is the wording correct here? Please let me know if it should be **consent (singular)**.

Reply:
Plural "Consents" is fine